components/ai_chat/resources/page/components/input_box/style.module.scss

```diff
@@ -21,19 +26,15 @@
   }
 }
 
-.textareaBox {
-  width: 100%;
-  font: var(--leo-font-default-regular);
-  color: var(--leo-color-text-primary);
-}
-
 .textarea {
   width: 100%;
-  padding: 10px 16px;
+  padding: 10px var(--leo-spacing-xl);
   resize: none;
   background-color: transparent;
   border: 0;
-
+  resize: none;
+  overflow: hidden;
+  grid-area: 1 / 1 / 2 / 2;
```

Comment:
should we be more explicit about applying styles to both `.textarea` and `.growWrap::after` so that we definitely apply any size-manipulating properties to both?

Reply:
fixed this with combined selectors